clients/admin-ui/src/features/common/PIIToggle.tsx

```diff
@@ -2,13 +2,13 @@ import { Switch } from '@fidesui/react';
 import React, { ChangeEvent } from 'react';
 import { useDispatch } from 'react-redux';
 
-import { setRevealPII } from './privacy-requests.slice';
+import { setRevealPII } from '../privacy-requests/privacy-requests.slice';
```

Comment:
I moved a few components into the common directory because they're being used on multiple pages.